### PR TITLE
fix: update permalink schema

### PR DESCRIPTION
**Pull request by eschutho**
### SUMMARY
Small fix for https://github.com/apache/superset/pull/24586 to  update the schema to point to the new one. The PostSchema no longer exists, so it was failing a test in the 2.1.1 release. 

### TESTING INSTRUCTIONS
The test in master isn't failing with the Post schema, likely because of some regression with openapi-spec-validator==0.5.6 that @villebro is looking into 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by michael-s-molina** [comment]
@eschutho Should we also update `docs/static/resources/openapi.json`?

```json
"ExplorePermalinkPostSchema": {
  "properties": {
    "formData": {
      "description": "Chart form data",
      "type": "object"
    },
    "urlParams": {
      "description": "URL Parameters",
      "items": {
        "description": "URL Parameter key-value pair",
        "nullable": true
      },
      "nullable": true,
      "type": "array"
    }
  },
  "required": ["formData"],
  "type": "object"
},
```

**Comment by eschutho** [comment]
> @eschutho Should we also update `docs/static/resources/openapi.json`?
> 
> ```json
> "ExplorePermalinkPostSchema": {
>   "properties": {
>     "formData": {
>       "description": "Chart form data",
>       "type": "object"
>     },
>     "urlParams": {
>       "description": "URL Parameters",
>       "items": {
>         "description": "URL Parameter key-value pair",
>         "nullable": true
>       },
>       "nullable": true,
>       "type": "array"
>     }
>   },
>   "required": ["formData"],
>   "type": "object"
> },
> ```

Yeah, good catch. Not sure if removing it is the right way to go, so lmk what you think. 🙏

File: docs/static/resources/openapi.json [hunk 1]
```diff
@@ -5454,25 +5454,6 @@
         },
         "type": "object"
       },
-      "ExplorePermalinkPostSchema": {
```
Comment (zephyring) [inline docs/static/resources/openapi.json:5457]: should you just rename this to `ExplorePermalinkPostSchema`? 
maybe just use `https://github.com/preset-io/superset/blob/master/superset/cli/update.py#L71` to update?

Comment (eschutho) [reply docs/static/resources/openapi.json:5457]: Nice, it looks like the cli command hasn't been run in a while. I've committed the output, if we think it's worth cleaning the whole file up in the process.

**Review by villebro — APPROVED** [review]
Thanks for fixing this @eschutho - I've indeed been looking into why our current OAS test isn't picking up this error, and it appears that regression is still there in the latest version of the validator library. I'll keep an eye out for a potential fix, and if one doesn't show up soon, I'll try to make a simple repro case and share it with the library devs.

**Review by michael-s-molina — APPROVED** [review]
LGTM. Thank you @eschutho!

**Comment by eschutho** [comment]
Thanks all!

**Pull request merged**